ext/ShapefileMakieExt.jl

```diff
@@ -10,7 +10,7 @@ GeoInterfaceMakie.@enable Shapefile.LinearRing
 Makie.plottype(tbl::Shapefile.Table) = Makie.plottype(Shapefile.shapes(tbl))
 Makie.plottype(shp::Shapefile.Handle) = Makie.plottype(Shapefile.shapes(shp))
 
-for T in (Any, Union{Type{Any}, Type{<:Makie.AbstractPlot}}, Type{<:Makie.Poly}, Type{<:Makie.Lines}, Makie.PointBased)
```

Comment:
Maybe a good change but unrelated?

Reply:
You mean in tests?

Reply:
Nope, somehow just when plotting in the REPL - at least on Makie 0.21.

Reply:
Thats a different PR then right? ;)

Reply:
Can do :D 
```suggestion
for T in (Any, Union{Type{Any}, Type{<:Makie.AbstractPlot}}, Type{<:Makie.Poly}, Type{<:Makie.Lines}, Makie.PointBased)
```